### PR TITLE
feat: add `(msb, getMsbD)_twoPow`

### PR DESCRIPTION
**Pull request by luisacicolini**

### CODE REVIEW

File: src/Init/Data/BitVec/Lemmas.lean [hunk 1]
```diff
@@ -2704,6 +2704,21 @@ theorem shiftLeft_eq_mul_twoPow (x : BitVec w) (n : Nat) :
...
+  simp [h₀, h₁] <;> omega
+
+@[simp]
+theorem msb_twoPow {i w: Nat} :
+    (twoPow w i).msb = (decide (i < w) && decide (i = w - 1)) := by
+  simp only [BitVec.msb, getMsbD_eq_getLsbD, Nat.sub_zero, getLsbD_twoPow,
+    Bool.and_iff_right_iff_imp, Bool.and_eq_true, decide_eq_true_eq, and_imp]
+  intros
+  omega
+
```
Comment (tobiasgrosser) [inline src/Init/Data/BitVec/Lemmas.lean:2721]: Can you move these below the getElem_twoPow theorems?

Comment (luisacicolini) [reply src/Init/Data/BitVec/Lemmas.lean:2721]: they are below already, no?

Comment (tobiasgrosser) [reply src/Init/Data/BitVec/Lemmas.lean]: Thank you.

**Review by tobiasgrosser — APPROVED** [review]

**Comment by tobiasgrosser** [comment]
I think this is ready to merge, @hargoniX.

**Pull request merged**